src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int MIN_NUMBER_OF_FIGURES = 0;
+    private static final int MAX_NUMBER_OF_FIGURES = 7;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[MIN_NUMBER_OF_FIGURES
+                + (int) (Math.random() * MAX_NUMBER_OF_FIGURES)];
```

Comment:
```suggestion
    private static final int MAX_NUMBER_OF_FIGURES = 7;

    public static void main(String[] args) {
        Figure[] figures = new Figure[(int) (Math.random() * MAX_NUMBER_OF_FIGURES)];
```

давай просто уберем эту переменную